robotbase/src/main/java/com/team2357/frc2022/arduino/RobotArduino.java

```diff
@@ -0,0 +1,40 @@
+package com.team2357.frc2022.arduino;
+
+import com.team2357.frc2022.Constants;
+import com.team2357.lib.arduino.ArduinoUSBController;
+
+public class RobotArduino extends ArduinoUSBController{
+
+    public RobotArduino(String ttyDevice) {
+        super(ttyDevice);
+        super.start();
+    }
+
+    public boolean getIntakeValue() {
+        boolean value = false;
+
+        if (super.isConnected()){
+            value = !super.getDeviceFieldBoolean(Constants.ARDUINO.INTAKE_SENSOR_JSON_NAME, Constants.ARDUINO.INTAKE_SENSOR_STATE_FIELD);
```

Comment:
Why are we looking for a `false` value here? Shouldn't the arduino be sending 'true' when the intake has a cargo present?

Reply:
Makes sense, we will make sure this change is reflected on the Arduino side.